<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Enhance</class>
 <widget class="QWidget" name="Enhance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图像增强</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/images/icon.png</normaloff>:/images/images/icon.png</iconset>
  </property>
  <widget class="QScrollArea" name="scrollArea1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>350</width>
     <height>350</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>348</width>
      <height>348</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="showPic1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="openPic">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>打开图像</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>280</y>
     <width>350</width>
     <height>350</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>348</width>
      <height>348</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="showPic2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_smooth">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>图像平滑</string>
   </property>
  </widget>
  <widget class="QComboBox" name="smooth">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>100</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>原图</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>均值滤波 (7*7)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中值滤波 (7*7)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>k近邻中值滤波 (7*7，k=24)</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_sharpen">
   <property name="geometry">
    <rect>
     <x>471</x>
     <y>140</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>图像锐化</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sharpen">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>140</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>原图</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>拉普拉斯算子</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sobel算子</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Prewitt算子</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>任意模板卷积，输入模板：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>80</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;-1 -1 -1&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;0 0 0&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;1 1 1&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="convolution">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>220</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>显示卷积结果</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>40</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>(行末必须换行，数字间只能有一个空格)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="homo">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>200</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>同态滤波</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
